src/main/java/order/Order.java

```diff
@@ -74,4 +74,14 @@ public HashMap<String, String> getOrderList() {
     public String getOrderStatus() {
         return this.orderStatus;
     }
+
+    /**
+     * Generate a random string represent as the order number of the order.
+     * @return A random generated string with digits and letters.
+     */
+    public static String generateOrderNum(){
```

Comment:
How do we know whether the order number already exists in the database for this function? The way I thought about how to solve it would be:
1. Whenever you generate an order number. Check all the order numbers in the database to ensure it is not repetitive.
2. Make a static variable called totalOrder, which will keep track of the total number of orders in our database, so the order number will always equal the current number of orders, so it avoids the prossibility of making a repetitive order number.